unafamilia/src/main/java/com/unfamilia/eggbot/domain/raidpackage/Item.java

```diff
@@ -50,4 +51,8 @@ public String getMedia() {
     public static Item findByName(String slug) {
         return find("slug", slug).firstResult();
     }
+
+    public static Item findById(Long id) {
```

Comment:
findById() and findByIdOptional() are inbuilt function, no need to redeclare it.

`var item = Item.<Item>findById(itemId);`

be mindful, that you need to define the type here otherwise it will be generalised into `PanacheEntityBase` type.

Reply:
Okay so I just need to completely remove this declaration? I'm not sure where you mean to define the type

Reply:
yes. in the method invocation, so exactly where you want to get an item by its id in the code.